tox.ini

```diff
@@ -73,9 +73,9 @@ commands = coverage erase
 skip_install = True
 parallel_show_output = True
 depends =
-    py311-latest{-pyuca,-noi18n}
-    py310-previous
-    py37-legacy
+    py312-django50-latest{-pyuca,-noi18n}
+    py311-django42-latest
+    py39-django32-latest
```

Comment:
These depends have to match the ones in envlist. A comment in envlist would be useful to ensure these are kept up to date when someone next updates them too.